src/metabase/automagic_dashboards/core.clj

```diff
@@ -496,25 +504,28 @@
         subform))
     {:type     :query
      :database (-> context :root :database)
-     :query    (cond-> {:source_table (if (->> context :source (instance? (type Table)))
+     :query    (cond-> {:source-table (if (->> context :source (instance? (type Table)))
                                         (-> context :source u/get-id)
                                         (->> context :source u/get-id (str "card__")))}
-                 (not-empty filters)
-                 (assoc :filter (->> filters
-                                     (map :filter)
-                                     (apply qp.expand/merge-filter-clauses)))
-
-                 (not-empty dimensions)
+                 (seq filters)
+                 (assoc :filter (vec
+                                 (cons
```

Comment:
wouldn't (apply vector :and) be clearer instead of relying on cons prepending to lists?

Reply:
Good point